docs/release/updating-to-osg-35.md

```diff
@@ -0,0 +1,286 @@
+Updating to OSG 3.5
+===================
+
+!!! note "OS Version Support"
+    OSG 3.5 only supports EL7
```

Comment:
Do we need to mention that we have 3.5 worker node support?

Reply:
Yeah, I think we should also update this as well https://github.com/opensciencegrid/docs/pull/783/files#diff-58913fbad905c5f06ee61cb671286ba10bfb69bb7a16cf93135b48695cf0ca7cR4 but figured it should go in a separate PR